anymail/backends/scaleway.py

```diff
@@ -0,0 +1,163 @@
+        )
+        self.region = get_anymail_setting(
+            "region", esp_name=esp_name, kwargs=kwargs, default="fr-par"
+        )
+
+        api_url = get_anymail_setting(
+            "api_url",
+            esp_name=esp_name,
+            kwargs=kwargs,
+            default=(
+                "https://api.scaleway.com/transactional-email/v1alpha1/regions/"
+                f"{self.region}"
+            ),
+        )
+        if not api_url.endswith("/"):
+            api_url += "/"
+        super().__init__(api_url, **kwargs)
+
+    def build_message_payload(self, message, defaults):
+        return ScalewayPayload(message, defaults, self)
+
+    def parse_recipient_status(self, response, payload, message):
+        parsed_response = self.deserialize_json_response(response, payload, message)
+        statuses = {}
+        try:
+            emails = parsed_response["emails"]
+        except (KeyError, TypeError):
+            raise AnymailAPIError(
+                "Invalid response from Scaleway API",
+                email_message=message,
+                payload=payload,
+                response=response,
+                backend=self,
+            )
+
+        for email_info in emails:
+            recipient = email_info.get("mail_rcpt")
+            message_id = email_info.get("id")
+            status = email_info.get("status")
+            status_map = {
+                "new": "queued",
+                "sending": "queued",
+                "sent": "sent",
+                "failed": "failed",
+                "canceled": "failed",
+            }
+            anymail_status = AnymailRecipientStatus(
+                message_id=message_id, status=status_map.get(status, "unknown")
+            )
+            if recipient:
+                statuses[recipient] = anymail_status
+        return statuses
+
+
+class ScalewayPayload(RequestsPayload):
+    def __init__(self, message, defaults, backend, *args, **kwargs):
+        self.project_id = backend.project_id
+        http_headers = kwargs.pop("headers", {})
+        http_headers["X-Auth-Token"] = backend.api_key
+        http_headers["Content-Type"] = "application/json"
+        super().__init__(
+            message, defaults, backend, headers=http_headers, *args, **kwargs
+        )
+
+    def get_api_endpoint(self):
+        return "emails"
+
+    def init_payload(self):
+        self.data = {
+            "to": [],
+            "cc": [],
+            "bcc": [],
+            "additional_headers": [],
+            "attachments": [],
+        }
+        if self.project_id:
+            self.data["project_id"] = self.project_id
+
+    def set_from_email(self, email):
+        self.data["from"] = {
+            "email": email.addr_spec,
+            "name": email.display_name or None,
+        }
+
+    def set_recipients(self, recipient_type, emails):
+        if emails:
+            self.data[recipient_type] = [
```

Comment:
```suggestion
        assert recipient_type in {"to", "cc", "bcc"}
        if emails:
            self.data[recipient_type] = [
```
(Just to double check Anymail's param still matches Scaleway's API payload field name.)